tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -792,9 +792,12 @@ internal static class FeatureFlags
 
             /// <summary>
             /// Enables injecting 128-bit trace ids into logs as a hexadecimal string.
+            /// If enabled, 64-bit trace ids will continue to be injected as decimal strings.
```

Comment:
I found the wording here a bit confusing, but realised it's because even if 128-bit generation is enabled, we'll have both 128 and 64 bit trace Ids, and this setting doesn't affect 64-bit IDs. Maybe the following makes that more obvious, by removing this line, and adding the one after?
```suggestion
            /// Enables injecting 128-bit trace ids into logs as a hexadecimal string.
```